### ISSUE TITLE
Contact form needs labels for accessibility

### ISSUE THREAD

**Issue by marcysutton** [johnotander/pixyll#341]
The contact form in your theme is missing labels to make them accessible to screen reader users. You can also increase the hit area for input fields by wrapping them with label elements. This should be an easy change that has a large user impact.

Problem code in [contact.html](https://github.com/johnotander/pixyll/blob/master/contact.html) (both form styles are affected):
```html
<input type="text" name="email" class="field-light" placeholder="{{ site.text.contact.email }}">
<textarea type="text" name="content" class="field-light" rows="5" placeholder="{{ site.text.contact.content }}" style="resize: vertical"></textarea>
<input type="text" name="_gotcha" style="display:none" />
<button type='submit' class="button button-blue button-big mobile-block">{{ site.text.contact.submit }}</button>
```

The easiest way to fix it for all users would be to use the text as a visual label, rather than a placeholder. You could also potentially use `aria-label` on the input and textarea itself, but this isn't as ideal since it only targets assistive technologies and is redundant with the placeholder.
```html
<label>
    {{ site.text.contact.email }}
  <input type="text" name="email" class="field-light" >
</label>
<label>
    {{ site.text.contact.content }}
    <textarea type="text" name="content" class="field-light" rows="5" style="resize: vertical"></textarea>
</label>
<input type="text" name="_gotcha" style="display:none" />
<button type='submit' class="button button-blue button-big mobile-block">{{ site.text.contact.submit }}</button>
```

**Comment by johnotander** [comment]
Thanks @marcysutton for the detailed issue and explanation. I've made your proposed updates by wrapping in relevant inputs with a label tag. Please let us know if you find anything else we can improve upon 😺
